### ISSUE TITLE
Blank force org on both categories

### ISSUE THREAD

**Issue by TrentBartlem** [BSData/adeptus-titanicus#87]
After you select a catalog (currently battlegroup v4 or household v3), there's a blank force entry, *then* the real one.

**Comment by capitaladot** [comment]
There is a requirement for the Game System to include a default force. Because the Forces both have Strategems that appeal to be distinct lists, my thinking they needed to reside in separate catalogs.
My initial fix idea is to add an anti-hoist character to peg the blank force from the GST to the bottom of the list. We could likewise mark it do not use or some similar text decoration.
Alternatively, perhaps there is another implementation idea that would avoid the blank fake force, since this game doesn't have the same default 40k style share Force pattern?

**Comment by TrentBartlem** [comment]
Some stratagems hide themselves if the allegiance doesn't match (Profane Blessing requires traitor allegiance etc). Could the same be done here? Or are strats just the tip of the iceberg

**Comment by capitaladot** [comment]
Unfortunately, I don't think there's a way to hide the blank stub force. I tried adding a hidden="true" attribute to no effect.

**Comment by capitaladot** [comment]
Played around a little. Confirmed that hiding the Force doesn't work (through the editor check box, a set Hidden modifier, or manually in the XML).
Tried hoist characters... no effect.
Tried moving Forces to the GST... broke dataset.

**Comment by Deviltree** [comment]
Can we make that an option to Create a Lance?  and maybe change Battlegroup to Titan Maniple?

**Comment by capitaladot** [comment]
> Can we make that an option to Create a Lance? and maybe change Battlegroup to Titan Maniple?

Unfortunately, that's not how the game system rules are put together; an army is composed of a Battlegroup (which in turn contains maniples, support titans, and banners) or (and perhaps and?; whether you can have multiple Battlegroups or Households, or both together is not addressed explicitly) Household, which must contain at least one Lance. 

Each Lance must contain at least three banners, two of which must be of the same type, and within each banner all knights must be of the same type and must have the same arm weapon options (but not upgrades like rocket pods, melta guns, grav guns, etc).

**Comment by capitaladot** [comment]
Battlescribe is sadly pretty opinionated in terms of what validation-related counting incrementors are available and where UI appears, and it is all tied to a Roster, which contains Forces, which contain Categories (like 40k FoC slots, or AT, and Entries (Shared or Root). 

Lance (awkwardly) needs to be an Entry instead of a Category, because it contains nested Banner options that are contained in each Lance that have internal validation constraints that apply to only that Lance, such as having to choose 3 Banners per Lance, and two of those Banners be of the same kind of Knight with the same arm weapons.

**Comment by capitaladot** [comment]
tl;dr the BS domain specific XML language is very limited.

**Comment by capitaladot** [comment]
I wonder if it would be more clear UI if I renamed the Lance Category to Household Lances?

**Comment by TrentBartlem** [comment]
> I wonder if it would be more clear UI if I renamed the Lance Category to Household Lances?

can't hurt

**Comment by TrentBartlem** [comment]
maybe we go back to a single force, and have a radio button to switch the list rules between household or battlegroup? I think that's what kill team does
